src/inspector_agent.cc

```diff
@@ -86,6 +86,37 @@ void StartIoInterrupt(Isolate* isolate, void* agent) {
   static_cast<Agent*>(agent)->StartIoThread();
 }
 
+std::string BuildMethodNotFoundErrorJSON(
+    int call_id, const std::string& method) {
+  auto error = protocol::DictionaryValue::create();
+  std::ostringstream message;
+  error->setInteger("code", -32601);  // Inspector magic number
+  message << '\'' << method << "\' wasn't found";
+  error->setString("message", message.str());
+  const auto& response = protocol::DictionaryValue::create();
+  response->setInteger("id", call_id);
+  response->setObject("error", std::move(error));
+  return response->toJSONString();
+}
+
+std::vector<std::string> Split(const std::string& string,
+                               const std::string& separator) {
```

Comment:
Any chance it makes sense to re-used `SplitString()` from `util.h`? It currently only supports single-char separators but that seems sufficient here